src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java

```diff
@@ -139,6 +139,46 @@ public void testLegacyAccountEntityToAttributes() {
 
     }
 
+    @Test
+    public void testGetCopy_typicalData_shouldCreateDeepCopy() {
+        AccountAttributes account = createValidAccountAttributesObject();
+
+        AccountAttributes copy = account.getCopy();
+
+        assertNotSame(account, copy);
+        assertNotSame(account, copy.studentProfile);
```

Comment:
you have a typo :P here `assertNotSame(account.studentProfile, copy.studentProfile);`